### DISCUSSION TITLE
Evaluation of ensemble NER models using SPACY Project Framework

### DISCUSSION THREAD

**Discussion by balachander1964** [explosion/spaCy#7746] [Help: Coding & Implementations] [answered]
Hi All, I will appreciate if you let me know how to evaluate multiple ner models using the Spacy 3 project framework. I am fine-tuning the SCISPACY BIONLP13CG and the BC5CDR models using the Flair datasets and trying to evaluate a combined BIONLP/CDR NER model. Awaiting your reply. Bala

**Comment by svlandeg** [comment, marked as answer]
Hi Bala,

To evaluate different spaCy `nlp` pipelines on the same gold-standard evaluation set, you can use [`nlp.evaluate`](https://spacy.io/api/language). You could also use the CLI command [`spacy evaluate`](https://spacy.io/api/cli#evaluate).

You could also have a look at https://github.com/explosion/projects/tree/v3/benchmarks for some example projects and inspiration. 

If you have more specific questions - shoot!
